packages/experience/src/pages/Consent/ScopeGroup/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { type Nullable } from '@silverhand/essentials';
+import classNames from 'classnames';
+import { useState, useCallback } from 'react';
+
+import DownArrowIcon from '@/assets/icons/arrow-down.svg';
+import CheckMark from '@/assets/icons/check-mark.svg';
+import { onKeyDownHandler } from '@/utils/a11y';
+
+import * as styles from './index.module.scss';
+
+type ScopeGroupProps = {
+  readonly groupName: string;
+  readonly isAutoExpand?: boolean;
```

Comment:
How about just `expanded` or `isExpanded`?

Reply:
`isAutoExpand` is ok here, since it reflects the business needs.